ebusd-0.5.x/ochsner/ote2.csv

```diff
@@ -0,0 +1,43 @@
+*r,ochsner,,,,15,0621,,,,,,,,,,,,,
+*w,ochsner,,,,15,0621,,,,,,,,,,,,,
+r,temperature,outside,Au�entemperatur,,,,00800040,,,parameter,,,,,,temperature,,,,
+r,temperature,water,Warmwasser Temperatur,,,,00840040,,,parameter,,,,,,temperature,,,,
+r,temperature,buffer,Vorlauftemperatur,,,,00e00040,,,parameter,,,,,,temperature,,,,
+r,heating,status,,,,,77800008,,,parameter,,,,status,,UCH,,,,
```

Comment:
the suffix doesn't seem to be of much relevance (in fact I have already observed the same commands with different suffixes, I just haven't figured out where that suffix comes from). nevertheless if I omit that suffix the request packet has a wrong payload length (this is not a broadcast message) and then the request fails.